lib/runtime/life/exports_test.go

```diff
@@ -14,6 +13,7 @@ import (
 	"github.com/ChainSafe/gossamer/lib/runtime/storage"
 	"github.com/ChainSafe/gossamer/lib/scale"
 	"github.com/ChainSafe/gossamer/lib/trie"
+	scale2 "github.com/ChainSafe/gossamer/pkg/scale"
```

Comment:
is the removal of the original `scale` in another issue?

Reply:
Yeah that would be in the integrate scale into runtime issue